emission/analysis/classification/inference/mode.py

```diff
@@ -298,8 +387,17 @@ def buildModelStep(self):
     model = forestClf.fit(self.selFeatureMatrix, self.cleanedResultVector)
     return model
 
+  def buildSpecificModel(self, algorithm_id):
```

Comment:
It isn't called yet. I added it because I knew we were eventually going to use different models to predict and so I wanted to have it there so I could incorporate it into the code now (but only one model would be supported at the moment) so it would be easier to adjust later.